applications/FluidDynamicsApplication/custom_processes/mass_conservation_check_process.h

```diff
@@ -0,0 +1,228 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    under supervision of Ruben Zorrilla
+//
+//
+
+#ifndef KRATOS_MASS_CONSERVATION_CHECK_PROCESS_H
+#define KRATOS_MASS_CONSERVATION_CHECK_PROCESS_H
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
```

Comment:
Check that you really need all these includes (same for the rest of files), I guess that you can avoid some of them.

Reply:
Many of them could be removed.

Reply:
please do this, unused includes only increase the compilation time